examples/distributed/primitives/example_get_block.py

```
@@ -0,0 +1,87 @@
+import tilelang.language as T
+import argparse
+import torch
+import torch.distributed as dist
+import torch.multiprocessing
+from tilelang.distributed.utils import init_dist
+
+tilelang.disable_cache()
+os.environ['NCCL_DEBUG'] = 'WARN'  # silence NCCL log
+
+
+def get_kernel(M, num_rank, block_M, threads):
```

Ruff (0.12.2)

Unused function argument: `num_rank`

(ARG001)